### ISSUE TITLE
Publish permission ignored allowing any authenticated user to publish

### ISSUE THREAD

**Issue by moll** [rlidwka/sinopia#238]
Hey,

Am I doing something wrong or is the `publish` permission totally ignored?
```
packages:
  "@*/*":
    access: $authenticated
    publish: none

  "*":
    access: $authenticated
    publish: none
```

I realize `none` isn't a special token, but both `none` and leaving `publish` empty still allows me to publish packages with `npm --registry example.com publish`.

**Comment by rlidwka** [comment]
Yep, I messed up with auth refactoring two weeks ago.

Fixed, added tests for it and published as `sinopia@1.2.2`. Thanks for reporting!

**Comment by moll** [comment]
Thank you!
